### PR TITLE
feat: Add naver auth button

### PR DESCRIPTION
**Pull request by KKimj**
image
image
image
image
image

### CODE REVIEW

**Comment by KKimj** [comment]
implement

https://github.com/elbeicktalat/flutter_auth_buttons/issues/63

File: lib/src/buttons/naver_auth_button.dart [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    super.onFocusChange,
+    super.focusNode,
+    super.autofocus,
+    super.text = 'Sign in with Naver',
+    @Deprecated(
+      'Use ThemeMode instead. '
+      'This property has no more effect. '
+      'This feature was deprecated after v3.0.0',
+    )
+    bool darkMode = false,
```
Comment (elbeicktalat) [inline lib/src/buttons/naver_auth_button.dart:41]: Since it's new widget there is no need to add this deprecated code.

**Review by elbeicktalat — CHANGES_REQUESTED** [review]
Nice job, only one fix.

Remove the Deprecated code.

File: lib/src/buttons/naver_auth_button.dart (hunk 1 above)
Comment (KKimj) [reply lib/src/buttons/naver_auth_button.dart]: Done!

**Review by elbeicktalat — APPROVED** [review]

**Pull request merged**